stable/nginx-ingress/Chart.yaml

```diff
@@ -1,6 +1,6 @@
 apiVersion: v1
 name: nginx-ingress
-version: 1.6.13
+version: 1.6.14
```

Comment:
Looks like your PR didn't get in before another one did. Bump the version to `1.6.15` and this should go through.

Reply:
Thanks! Amended the commit bumping the version